app/views/plan/index.scala

```diff
@@ -290,7 +290,9 @@ object index {
         ),
         dt(otherMethods()),
         dd(
-          lichessIs(a(href := "https://causes.benevity.org/causes/250-5789375887401_bf01")(registeredWithBenevity())),
+          lichessIsRegisteredWith(),
+          a(href := "https://causes.benevity.org/causes/250-5789375887401_bf01")("Benevity"),
+          ".",
```

Comment:
this is still translation lego. In some languages, "Benevity" should not be at the end of the sentence.

It should be:
```scala
lichessIsRegisteredWith(a(href := "https://causes.benevity.org/causes/250-5789375887401_bf01")("Benevity"))
```
```xml
<string name="lichessIsRegisteredWith">Lichess is registered with %s.</string>
```

Reply:
Sorry, I guess I didn't quite understand that. I hope the next commit is satisfactory :D